public/app/features/alerting/unified/Admin.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { Button, ConfirmModal } from '@grafana/ui';
+import { useAlertManagerSourceName } from './hooks/useAlertManagerSourceName';
+import { AlertingPageWrapper } from './components/AlertingPageWrapper';
+import { AlertManagerPicker } from './components/AlertManagerPicker';
+import { GRAFANA_RULES_SOURCE_NAME } from './utils/datasource';
+import { useDispatch } from 'react-redux';
+import { deleteAlertManagerConfigAction } from './state/actions';
+import { useUnifiedAlertingSelector } from './hooks/useUnifiedAlertingSelector';
+
+export default function Admin(): JSX.Element {
+  const dispatch = useDispatch();
+  const [alertManagerSourceName, setAlertManagerSourceName] = useAlertManagerSourceName();
+  const [showConfirmDeleteAMConfig, setShowConfirmDeleteAMConfig] = useState(false);
+  const { loading } = useUnifiedAlertingSelector((state) => state.deleteAMConfig);
+
+  const resetConfig = () => {
+    if (alertManagerSourceName) {
+      dispatch(deleteAlertManagerConfigAction(alertManagerSourceName));
+    }
+    setShowConfirmDeleteAMConfig(false);
+  };
+
+  return (
+    <AlertingPageWrapper pageId="alerting-admin">
+      <AlertManagerPicker current={alertManagerSourceName} onChange={setAlertManagerSourceName} />
+      {alertManagerSourceName && (
+        <>
+          <Button disabled={loading} variant="destructive" onClick={() => setShowConfirmDeleteAMConfig(true)}>
+            Reset alert manager configuration
+          </Button>
+          {!!showConfirmDeleteAMConfig && (
+            <ConfirmModal
+              isOpen={true}
+              title="Reset alert manager configuration"
+              body={`Are you sure you want to reset configuration ${
```

Comment:
I think it may be worth calling out what the consequences of resetting the alertmanager config are. Also, is it `Alertmanager` or `Alert manager`? Prometheus uses `Alertmanager` as one word.

Reply:
dam, good point! Added note about consequences and aligned with docs throught the ui, "alert manager" -> "Alertmanager".